docs/data/stashcache/install-cache.md

```diff
@@ -57,13 +57,13 @@ statistics about the cache. To simplify installation, OSG provides convenience R
 software with a single command:
 
     :::console
-    root@host # yum install stashcache-cache-server
+    root@host # yum install --enablerepo=osg-development stashcache-cache-server
```

Comment:
Ah, enabling osg-development in the docs!

Reply:
Yup - not much choice here as the doc content matches osg-development!